exercises/transpose/package.yaml

```diff
@@ -0,0 +1,20 @@
+name: transpose
+version: 1.0.0.1
```

Comment:
Because https://github.com/exercism/problem-specifications/blob/master/exercises/transpose/canonical-data.json#L3 says 1.1.0, we should say 1.1.0.1 here.

Reply:
Ah, nice find. I forgot to check that.